tasks/changelog.rb

```diff
@@ -1,13 +1,30 @@
 begin
   require "github_changelog_generator/task"
 
-  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
-    config.issues = false
+  # Take the current changelog and move it to HISTORY.md. Should be done when
+  # cutting a release
+  task :archive_changelog do
+    changelog = File.readlines("CHANGELOG.md")
+    File.open("HISTORY.md", "w+") { |f| f.write(changelog[2..-1].join("")) }
+  end
+
+  # Run this to just update the changelog for the current release. This will
+  # take what is in History and generate a changelog of PRs between the most
+  # recent tag in HISTORY.md and HEAD.
+  GitHubChangelogGenerator::RakeTask.new :update_changelog do |config|
     config.future_release = Chef::VERSION
+    config.between_tags = ["v#{Chef::VERSION}"]
+    config.max_issues = 0
```

Comment:
Right now it shows Closed issues, but they are grouped together and we can easily clean them out manually or (ideally) with automation. Also, I'm sure with a minor PR upstream we could fix it properly.